samples/go/csv/csv-import/perf_test.go

```diff
@@ -50,29 +46,57 @@ func (s *perfSuite) Test01ImportSfCrimeBlobFromTestdata() {
 	defer s.closeGlob(raw)
 
 	blob := types.NewBlob(io.MultiReader(raw...))
-	fmt.Fprintf(s.W, "csv/raw is %s\n", humanize.Bytes(blob.Len()))
+	fmt.Fprintf(s.W, "\tsf-crime is %s\n", humanize.Bytes(blob.Len()))
 
-	ds := dataset.NewDataset(s.Database, "csv/raw")
-	_, err := ds.CommitValue(blob)
-	assert.NoError(err)
+	// CommitValue is much faster the first time, so pause the test while it's happening.
+	s.Pause(func() {
```

Comment:
per meatspace: this seems a bit strange to me. it might be better to leave this time in and then have a way in the UI to show "hot" vs "cold" perf.